CMakeLists.txt

```diff
@@ -24,7 +24,7 @@ option(ENABLE_OPENMP "Enable OpenMP flags" OFF)
 option(ENABLE_BOOST  "Enable Boost libraries" OFF)
 option(NALU_WIND_SAVE_GOLDS  "Save gold files to directory when running tests" OFF)
 
-set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
isn't C++17 already turned on?  does this just need a rebase or merge from master?

Reply:
yeah it hasn't been updated since we merged #1036